<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Traukinys</class>
 <widget class="QWidget" name="Traukinys">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>450</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>300</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>320</width>
          <height>300</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>130</y>
           <width>51</width>
           <height>17</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Atgal</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bStabis">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>220</y>
           <width>81</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(214, 0, 0);
background-color: rgb(255, 170, 73)</string>
         </property>
         <property name="text">
          <string>STABDIS</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="pSignalas">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>191</width>
           <height>23</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-91</number>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>-90</number>
         </property>
         <property name="format">
          <string>%v dBm</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>66</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Signalas:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>130</y>
           <width>66</width>
           <height>17</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Pirmyn</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Traukinys:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
        <widget class="QLabel" name="lSpeed">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>140</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 italic 12pt sans;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>100</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 italic 20pt sans;</string>
         </property>
         <property name="text">
          <string>Greitis:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
        <widget class="QDial" name="dGreitis">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>161</width>
           <height>151</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgba(0, 25, 255, 128)</string>
         </property>
         <property name="minimum">
          <number>-100</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="stopButton">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>100</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 1000 10pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QToolButton" name="buttonRemote">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>10</y>
           <width>27</width>
           <height>26</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Nuotilinis valdymas</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDockWidget" name="dockWidget">
        <property name="windowTitle">
         <string>Info</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents_2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPlainTextEdit" name="infoText">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;monospace&quot;;</string>
            </property>
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="plainText">
             <string/>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
